maven project/buy-a-car/src/main/java/com/retail/shop/car/App.java

```diff
@@ -19,14 +25,14 @@ public class App
 {
     public static void main( String[] args ) throws StreamReadException, DatabindException, IOException
```

Comment:
Lets make it dynamic based on User input - Use Scanner class to read the value from the console - I mean prompt the user to enter the input. Say something like below:

Console:
===================

Welcome to XYZ Car dealer. Please select from the below option to continue:

1. Check Car Inventory
2. Add New Car
3. Remove Existing Car
4. Update Car Details
5. Order New Car

Consider option 1 is doable now - So on providing "1"; User should be able to  view all cars. And any other option should be throw as exception, preferably a customised exception.
Refer here: https://www.javatpoint.com/exception-handling-in-java